Integrations/integration-Aella_StarLight.yml

```diff
@@ -0,0 +1,331 @@
+            #demisto.info('Data is {}'.format(data))
+            if not isinstance(data, dict):
+                demisto.error('Data returned in wrong format {}'.format(data))
+                demisto.incidents([])
+                return
+            hits = data.get('hits', {}).get('hits', [])
+            incidents = []
+            for hit in hits:
+                source = hit.get('_source', None)
+                if not source:
+                    continue
+                event_name = source.get('event_name', None)
+                try:
+                    event_severity = int(source.get('severity', None))
+                    if event_severity > 75:
+                        severity = 'high'
+                    elif event_severity > 50:
+                        severity = 'medium'
+                    else:
+                        severity = 'low'
+                except:
+                    severity = 'unknown'
+
+                if not event_name:
+                    continue
+                eid = hit['_id']
+                sdi = '{}_{}'.format(event_name, eid)
+                incident = {
+                    'name' : sdi,
+                    'label' : 'Starlight event',
+                    'aella_eid' : eid,
+                    'aella_event' : event_name,
+                    'evnet_severity': severity,
+                    'rawJSON':json.dumps({  'name':sdi,
+                                            'label':'Starlight event',
+                                            'aella_eid': eid,
+                                            'aella_event':event_name,
+                                            'evnet_severity': severity
+                                        })
+                }
+                incidents.append(incident)
+
+            demisto.info('Incidents is {}'.format(incidents))
+            demisto.incidents(incidents)
+            demisto.info('Set LastRun to be {}'.format(cur_t))
+            demisto.setLastRun(cur_t)
+        else:
+            demisto.info('Poll incidents failed {}'.format(action_result))
+            demisto.incidents([])
+
+    def aella_get_event_command():
+        demisto.info('Aella started get-event with {}'.format(demisto.args()['eid']))
+        event_id = demisto.args()['eid']
+        query_json = { 'query': { 'match': { '_id': event_id } } }
+        end_point = URL + '/{0}/{1}/_search'.format('aella-ser*', 'amsg')
+
+        action_result = {}
+        make_rest_call(end_point,
+            USERNAME, PASSWORD, action_result, data=query_json
+        )
+        demisto.info('REST API query eid returns {}'.format(action_result))
+        if action_result['status'] == 'Success':
+            demisto.info('Run Query is successful')
+            response = action_result.get('data')
+            timed_out = response.get('timed_out', False)
+            hits = response.get('hits', {}).get('hits', [])
+            for item in hits:
```

Comment:
The `location_geo` field does not return well to the war room - missing some spacing, please handle that.

Reply:
This should be handled by the common server code. @yaakovi